src/index.js

```diff
@@ -42,12 +42,19 @@ export default function Loadable<Props: {}, Err: Error>(opts: Options) {
   let outsidePromise = null;
   let outsideError = null;
 
-  if (serverSideRequirePath) {
+  if (!isWebpack && serverSideRequirePath) {
     outsideComponent = tryRequire(serverSideRequirePath);
   }
 
   if (isWebpack && webpackRequireWeakId) {
-    outsideComponent = tryRequire(webpackRequireWeakId());
+    let weakId = webpackRequireWeakId();
+    if (__webpack_modules__[weakId]) {
```

Comment:
wouldn't the `isWebpack` above be sufficient? `__webpack_require__` and `__webpack_modules__ ` are both internals of a webpack runtime. Otherwise `let isWebpack = typeof __webpack_require__ !== "undefined" && typeof __webpack_modules__ !== "undefined";` might be neater?